prom2teams/teams/template.j2

```diff
@@ -3,7 +3,8 @@
     'resolved' : '2DC72D',
     'critical' : '8C1A1A',
     'severe' : '8C1A1A',
-    'warning' : 'FF9A0B'
+    'warning' : 'FF9A0B',
+    'unknown' : '0066CC'
```

Comment:
Could you use 'CCCCCC' instead of '0066CC'? Probably a grey color suggest that is an error (better than blue).

Reply:
OK, will fix.